src/app/src/androidTest/java/com/example/bookmark/mocks/MockStorageService.java

```diff
@@ -23,12 +23,24 @@ public static StorageService getMockStorageService() {
         List<Book> books = new ArrayList<>();
         MockModels.getMockBook1().setStatus(Book.Status.REQUESTED);
         MockModels.getMockBook2().setStatus(Book.Status.REQUESTED);
+        MockModels.getMockBook4().setStatus(Book.Status.BORROWED);
+        MockModels.getMockBook5().setStatus(Book.Status.ACCEPTED);
+        MockModels.getMockBook6().setStatus(Book.Status.BORROWED);
```

Comment:
Modifying a mock is not safe. You must set the book status in the MockModels class. Not here. Otherwise, our other tests are going to fail.